### PR TITLE
DT-1284: 🎨 Remove ejs, sinon & chai

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "jsx-a11y/control-has-associated-label": 0,
     "react/no-deprecated": 0,
     "react/jsx-fragments": 0,
+    "no-only-tests/no-only-tests": "error",
   },
-  "plugins": ["prettier"]
+  "plugins": ["prettier", "no-only-tests"],
```
Comment (petergphillips) [inline .eslintrc:33]: changes copied from psh

File: backend/index.js [hunk 2]
```diff
@@ -91,10 +90,6 @@ app.use(bodyParser.json())
 app.use(setupSass())
 app.use(setupStaticContent())
 
-app.get('/terms', async (req, res) => {
```
Comment (petergphillips) [inline backend/index.js:94]: not used as we use the support app for terms these days

File: backend/tests/sessionManagementRoutes.test.js [hunk 3]
```diff
@@ -2,17 +2,10 @@
 const request = require('supertest')
 const express = require('express')
 const bodyParser = require('body-parser')
-const cookieSession = require('cookie-session')
```
Comment (petergphillips) [inline backend/tests/sessionManagementRoutes.test.js:5]: file copied from psh

**Review by JamieDraperUK — APPROVED** [review]
LGTM

**Pull request merged**